website/blog/2019-06-04-hello-reason-react-native.md

```diff
@@ -0,0 +1,129 @@
+
+We knew that we wanted to:
+
+- offer to `bs-react-native` users a migration plan, so they can start very
+  easily to use JSX 3 and new `reason-react` (so they can use hooks),
+- offer new `reason-react-native` bindings, more accessible and with a zero cost
+  approach.
+- offer a smooth transition between these two packages so people can slowly
+  upgrade their codebase without having to make an insane PR that switches from
+  one to the other.
+
+We don't want your co-workers to hate you with a PR that comes directly from
+hell 😅.
+
+That's where we thought about offering `bs-react-native-jsx3-compat`. It's a
+package that has the same APIs has `bs-react-native` but that uses
+`reason-react-native` under the hood.
+
+**This means you will have to make a migration from JSX 2 to 3**. That's the
+only thing you will have to do, then with a single change from `bs-react-native`
+with `bs-react-native-jsx3-compat` in your `bs-dependencies` you will be ready
+to use hooks and directly benefit of (almost) zero-cost bindings!
+
+And since we care about you and don't want you to upgrade from JSX 2 to 3 by
+hand, we got you covered with a script made with love to help you with that.
+
+[You will find all the required instructions to migrate from `bs-react-native` to `bs-react-native-jsx3-compat` in our documentation](/reason-react-native/en/docs/migration/jsx3/).
+
+## tl;dr
+
+- `bs-react-native` has now entered a frozen state and won't receive any upgrade
+- `bs-react-native-jsx3-compat` allows you to use existing code based on
+  `bs-react-native` with `reason-react` 0.7.0 and JSX 3
+- `reason-react-native` package contains the new bindings for React Native, that
+  offers more accessible APIs and that is almost zero-cost
+- [We offer you an easy and smooth migration plan so you can start using `reason-react-native` and slowly say bye to `bs-react-native`](/reason-react-native/en/docs/migration/jsx3/).
+
+## Thanks
+
+This required effort has been keeping us busy for weeks and wouldn't have been
+possible without the following people:
+
+- CCA team and especially @cknitt for leading us to the right direction for the
+  new zero-cost bindings and all the contributions that goes with this move
+- @sgny for all his contributions to the new bindings
+- @MoOx for leading the effort, the compat layer and the new website you are
+  seeing here
+- @bloodyowl for his magical
+  [reason-react-compat](https://github.com/bloodyowl/reason-react-compat) layer
+  and
+  [the alternate JSX 3 upgrade path](https://bloodyowl.github.io/blog/2019-04-19-an-alternative-migration-path-for-reason-react/)
+  that goes with it
+- all the people that helped us on Discord and GitHub in various code reviews
+  and improvements like the react native template (thanks @dawee)
+
+## What's next?
+
+Now that `reason-react-native` is ready to be used and that our migration plan
+is usable, we will probably be focused documentation as lots of APIs and
```

Comment:
`we will probably be focused on documentation`

Reply:
`as lots of APIs and Components need to be documented.` -> `in order to offers the best developer experience possible`